<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtLifecycleClass</class>
 <widget class="QWidget" name="QtLifecycleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtLifecycle</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>subpage</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>831</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid blue;</string>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>subpage2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="printstack">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>print stack</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtLifecycle.qrc"/>
 </resources>
 <connections/>
</ui>
